### ISSUE TITLE
Move `Timeline` to separate package

### ISSUE THREAD

**Issue by thyhjwb6** [Royal-Navy/design-system#3030]
Timeline should be moved out.

People are quite tightly coupled to certain implementations of that.

And it prevents them getting component updates.

**Comment by m7kvqbe1** [comment]
- Move the Timeline into a package so that it can be consumed independent of the rest of the Component Library:
  - __Option 1:__ Keep within the monorepo and maintain version parity
    - Allows the consumer to pin to an earlier release
    - It's not clear from the tag which packages have changed (applies to all packages)
    - Most value for least effort
    - Lerna has a new maintainer
    - New package would start from current HEAD
  - __Option 2:__ Move into it's own respository and version independently
    - Negates the benefits of a monorepo (shared CI, maintenance overhead, consuming packages in dev)
  - __Option 3:__ Keep within monorepo and version independently but move away from Lerna + Yarn workspaces
    - Grant previously struggled to get the current setup to version packages independently
    - Lerna has issues with maintainers: https://github.com/lerna/lerna/issues/3121
    - Seperate larger piece of work?

**Comment by c9qsm62x** [comment]
Originally this was an issue when there was active development on the timeline component.  We faced issue when scaling was released as a MVP and there was missing features such as opt-out and min and max level zoom. So we waited until some issues were fixed before updating the library. The timeline component is the most complex feature in our app and sometimes takes a bit of time to fix when updating the mod uk DS lib.  

I would welcome this if development resumed on the timeline component. 

I also think this will benefit the design system team to make further enhancement that are breaking changes, and not worry too much about the downstream application.
